vendor/k8s.io/test-infra/prow/github/fakegithub/fakegithub.go

```diff
@@ -1108,6 +1108,10 @@ func (f *FakeClient) ListCurrentUserOrgInvitations() ([]github.UserOrgInvitation
 	return ret, nil
 }
 
+func (f *FakeClient) QueryWithGitHubAppsSupport(ctx context.Context, q interface{}, vars map[string]interface{}, org string) error {
+	return nil
+}
+
```

Comment:
You cannot simply change files under `vendor`: this is upstream code we vendor. See https://go.dev/ref/mod#vendoring for more details. If you need this change, you have two options:

1) Wrap the `FakeClient` in a downstream struct and add the missing method there:

```go
type MyFakeClient FakeClient
func (f *MyFakeClient) QueryWithGitHubAppsSupport(...
```

2) Add the change to https://github.com/kubernetes/test-infra/blob/master/prow/github/fakegithub/fakegithub.go by filing a PR to k/test-infra, and after it merges, file a PR here to bump the vendoring.

The latter is the *rght* way but it's gonna take a bit longer (especially to this feature branch) so I'd personally go with the former.